frontend/src/modalNewData.js

```diff
@@ -0,0 +1,203 @@
+import React, {useState, useEffect} from "react";
+import ReactDOM from "react-dom";
+
+type modalProp = {
+    openModal: Boolean,
+    onClose: object,
+    newData: Array,
+    importNewData: object
+}
+
+type NewDataItem_type = {
+    data: object,
+    index: integer
+}
+
+const NewDataItem = (props : NewDataItem_type) => {
+    return (
+        <li className='list-group-item checkbox'>
+            <div className="form-check form-check-reverse">
```

Comment:
I **know** I can be irritating with formatting stuff, but can we be consistent with quotes? I would use double-quotes for HTML attribute values.

To be fair: I need to add a linter in next version of the app (as black on Python side).